ext/nnstreamer/tensor_converter/tensor_converter_flexbuf.cc

```diff
@@ -116,11 +116,14 @@ flxc_convert (GstBuffer *in_buf, gsize *frame_size, guint *frames_in, GstTensors
   GstMemory *in_mem, *out_mem;
   GstMapInfo in_info;
   guint mem_size;
-  gpointer mem_data;
+  gsize buf_size = gst_buffer_get_size (in_buf);
 
   in_mem = gst_buffer_peek_memory (in_buf, 0);
+  /** Use input memory as output memory to avoid memory copy */
+  gst_memory_ref (in_mem);
```

Comment:
how about moving this to line 139, line 135 also handles error case.

Reply:
Right. Updated.